package.json

```diff
@@ -14,12 +14,19 @@
     "semantic-ui-react": "0.71.5"
   },
   "devDependencies": {
+    "enzyme": "^2.9.1",
+    "enzyme-to-json": "^1.6.0",
     "eslint": "4.3.0",
     "eslint-config-airbnb": "15.1.0",
     "eslint-plugin-import": "2.7.0",
     "eslint-plugin-jsx-a11y": "5.1.1",
     "eslint-plugin-react": "7.1.0",
-    "react-scripts": "1.0.10"
+    "nock": "^9.0.14",
```

Comment:
@Sanderson239 please remove this `^` as well as the others in this file, thanks!